src/observable/ArrayLikeObservable.ts

```diff
@@ -55,7 +55,8 @@ export class ArrayLikeObservable<T> extends Observable<T> {
 
   protected _subscribe(subscriber: Subscriber<T>): TeardownLogic {
     let index = 0;
-    const { arrayLike, scheduler } = this;
+    const arrayLike = this.arrayLike;
+    const scheduler = this.scheduler;
```

Comment:
This is for to avoid a [TypeScript compiler's bug](https://github.com/Microsoft/TypeScript/issues/11324) (It will be [fixed in TS2.1](https://github.com/Microsoft/TypeScript/pull/11998)).